src/components/NcActions/NcActions.vue

```diff
@@ -1094,6 +1094,18 @@ export default {
 
 			default: null,
 		},
+
+		/**
+		 * Specify the button size
+		 * Accepted values: `'small'`, `'normal'` (default), `'large'`
+		 */
```

Comment:
Styleguidist parses `includes` to see values. And in TS we'll specify it explicitly. So we don't need to describe it in the documentation.

<img width="855" height="104" alt="image" src="https://github.com/user-attachments/assets/ecfaff0f-ec97-4352-a1a9-7a1d5347b42e" />


And I'd mention that this is specifically inline buttons and the trigger size, not "entire NcActions menu size".


```suggestion
		/**
		 * Trigger and inline button sizes
		 */
```

Reply:
What about the second part? :)

Reply:
Overlooked